File: src/main/java/com/hy/frame/camera/CameraFocusListener.java
package com.hy.frame.camera;

/**
 * CameraFocusListener
 *
 * @author HeYan
 * @time 2017/4/28 15:33
 */
public interface CameraFocusListener {
    void onFocusBegin(float x, float y);

    void onFocusEnd();
}

File: Frame/src/main/java/com/hy/frame/camera/CameraFocusListener.kt
package com.hy.frame.camera

/**
 * CameraFocusListener
 * @author HeYan
 * @time 2017/4/28 15:33
 */
interface CameraFocusListener {
    fun onFocusBegin(x: Float, y: Float)

    fun onFocusEnd()
}
